startup.sh

```diff
@@ -11,19 +15,27 @@ __     __        _  __         _   _       _        ____  ___
 EOF
 tput sgr0
 
-# Generate PKI and config if necessary
-if test ! -d data; then
-  command -v cfssl >/dev/null || brew install cfssl
-  ./generate/hub-dev-pki.sh
+# Build a docker image with all of our dependencies for future steps
+if ! command -v docker >/dev/null; then
+  >&2 echo "docker: command not found. verify-local-startup requires docker."
+  exit 1
 fi
 
+docker build -t verify-local-startup .
+
+docker run -t -v "$script_dir:/verify-local-startup/" verify-local-startup -c '
+set -e
```

Comment:
This block should be in a separate script, ideally.

Reply:
Added a comment explaining why it's inlined.

Reply:
It being another script would be nice for localising this, but it might be confusing: e.g. if there was:
- `startup.sh`
- `docker-startup.sh`

where `startup.sh` calls `docker-startup.sh` inside a docker run block, that would be quite confusing, having it inline is probably least worst?